sdk/core/core-http/src/serviceClient.ts

```diff
@@ -775,6 +783,12 @@ export function createPipelineFromOptions(
     requestPolicyFactories.push(proxyPolicy(pipelineOptions.proxyOptions));
   }
 
+  if (clientRequestIdOptions.shouldGenerateClientRequestId) {
```

Comment:
do we want to change the order of this policy? I don't think it'll matter, but I was a little curious why this moved up.

Reply:
Yeah, I'll put it back in the same place it was before